package.json

```diff
@@ -7,6 +7,7 @@
         "clean": "del ./dist",
         "cpy": "cpy",
         "build": "npm-run-all clean build:angular webpack build:icons",
+        "postbuild": "node ./scripts/aot.spec.js",
```

Comment:
If this fails, does the `npm run build` still return with an error state? I would expect, but wanted to be certain.

Reply:
I can rename.

Reply:
Yes. It was erroring out on the build and npm was exiting w/ error state.